build.gradle

```diff
@@ -77,6 +77,20 @@ test {
         excludeTestsMatching '*SmokeTest'
         excludeTestsMatching '*WIP*'
     }
+
+    testLogging {
```

Comment:
Good call 👍

Reply:
Thanks. Had been bothering me for a while :-)